yarn.lock

```diff
@@ -7180,6 +7180,13 @@ read-pkg@^3.0.0:
     normalize-package-data "^2.3.2"
     path-type "^3.0.0"
 
+read@^1.0.7:
```

Comment:
I don't think we're using read anymore. May want to revert the lock file.

Reply:
Good call.